scripts/solana_verify_cpi_call.ts

```diff
@@ -0,0 +1,118 @@
+/*
+  This script fetchs a key/value pair and its proofs from the Umbrella API and
+  validated them with a call to the `caller` Solana program, this last performs
+  a cross-program invocation (CPI) to the official chain program for the final
+  validation. The Merkel root hash is already stored in a block which is a
+  Solana data account referenced with a Program Derived Account (kind of
+  public-key), this last can is derived in our program using a the block ID and
+  the official program ID of `chain`.
+ */
+
+import * as anchor from '@project-serum/anchor';
+import { Program } from '@project-serum/anchor';
+import { PublicKey, SystemProgram } from '@solana/web3.js';
+import { LeafKeyCoder, LeafValueCoder } from '@umb-network/toolbox';
+import axios from 'axios';
+import 'dotenv/config';
+
+const {API_BASE_URL, API_KEY} = process.env;
+
+const chainId = "4SPgs3L7Ey9VyRuZwx4X3y86LSAZXP2Hhpz9Sps4v3iT";
+const callerId = "BmmRtz8Zf4rjQgWT643QG2eqHVkXzebSsnR7XipFTrAg";
+
+const IDL = JSON.parse(
+  require("fs").readFileSync("./artifacts/solana-idl/chain.json", "utf8")
```

Comment:
instead if putting require("fs") everywhere, you can make a single import at the top of the file.
Also, please use a ' instead of "